docs/readthedocs/source/doc/LLM/Quickstart/install_windows_gpu.md

```diff
@@ -0,0 +1,96 @@
+
+* After installation, open the **Anaconda Prompt**, create a new python environment `llm`:
+  ```bash
+  conda create -n llm python=3.9 libuv
+  ```
+* Activate the newly created environment `llm`:
+  ```bash
+  conda activate llm
+  ```
+ 
+## Install oneAPI 
+
+* With the `llm` environment active, use `pip` to install the **OneAPI Base Toolkit**:
+  ```bash
+  pip install dpcpp-cpp-rt==2024.0.2 mkl-dpcpp==2024.0.0 onednn==2024.0.0
+  ```
+  
+## Install `bigdl-llm`
+
+* With the `llm` environment active, use `pip` to install `bigdl-llm` for GPU: 
+  ```bash
+  pip install --pre --upgrade bigdl-llm[xpu] --extra-index-url https://pytorch-extension.intel.com/release-whl/stable/xpu/cn/
+  ```
+  > Note: If there are network issues when installing IPEX, refer to [this guide](https://bigdl.readthedocs.io/en/latest/doc/LLM/Overview/install_gpu.html#install-bigdl-llm-from-wheel) for more details. 
+
+* You can verfy if bigdl-llm is successfully by simply importing a few classes from the library. For example, in the Python interactive shell, execute the following import command:
+  ```python
+  from bigdl.llm.transformers import AutoModel,AutoModelForCausalLM
+  ```
+
+## A quick example
+* Next step you can start play with a real LLM. We use [phi-1.5](https://huggingface.co/microsoft/phi-1_5) (an 1.3B model) for demostration. You can copy/paste the following code in a python script and run it. 
+> Note: to use phi-1.5, you may need to update your transformer version to 4.37.0.  
+> ```
+> pip install -U transformers==4.37.0 
+> ```
+> Note: when running LLMs on Intel iGPUs for Windows users, we recommend setting `cpu_embedding=True` in the from_pretrained function.
+> This will allow the memory-intensive embedding layer to utilize the CPU instead of iGPU.
+
+   ```python
+   import torch
+   from bigdl.llm.transformers import AutoModelForCausalLM
+   from transformers import AutoTokenizer, GenerationConfig
+   generation_config = GenerationConfig(use_cache = True)
+   
+   tokenizer = AutoTokenizer.from_pretrained("microsoft/phi-1_5", trust_remote_code=True)
+   # load Model using bigdl-llm and load it to GPU
+   model = AutoModelForCausalLM.from_pretrained(
+       "microsoft/phi-1_5", load_in_4bit=True, cpu_embedding=True, trust_remote_code=True)
+   model = model.to('xpu')
+
+   # Format the prompt
+   question = "What is AI?"
+   prompt = " Question:{prompt}\n\n Answer:".format(prompt=question)
+   # Generate predicted tokens
+   with torch.inference_mode():
+       input_ids = tokenizer.encode(prompt, return_tensors="pt").to('xpu')
+       output = model.generate(input_ids, do_sample=False, max_new_tokens=32, generation_config = generation_config).cpu()
+       output_str = tokenizer.decode(output[0], skip_special_tokens=True)
+       print(output_str)
+   ```
+
+* An example output on the laptop equipped with i7 11th Gen Intel Core CPU and Iris Xe Graphics iGPU looks like below. 
```

Comment:
How to run this? And any ENV needed (see https://bigdl.readthedocs.io/en/latest/doc/LLM/Overview/install_gpu.html#runtime-configuration)

Reply:
Copy/paste the python code and run it in the environment created above. When using `pip install dpcpp-cpp-rt==2024.0.2 mkl-dpcpp==2024.0.0 onednn==2024.0.0` to install OneAPI, we do not need to add runtime configurations.

Reply:
1) Add instructions
2) I think `SYCL_CACHE_PERSISTENT` and `BIGDL_LLM_XMX_DISABLED` need to be configured differently for different GPU @Oscilloscope98 ?

Reply:
Yes, we have different configuration for different GPU. Detailed are in here: https://bigdl.readthedocs.io/en/latest/doc/LLM/Overview/install_gpu.html#runtime-configuration

Reply:
submitted a new PR to address these revisions:
- quick example instructions step-by-step
- add run time configuration step
- add us/cn extra index url specs